RevenueCatUI/Templates/V2/ViewHelpers/Fill.swift

```diff
@@ -56,4 +56,28 @@ extension Shape {
     }
 }
 
+extension UnitPoint {
+
+    init(angle: Angle) {
```

Comment:
We're still using the old way of calculating the gradient but only for the Timeline Component connectors because these use a different way of configuring the gradient using a Shape where we don't have the size of the rectangle precomputed. This shouldn't be a problem since connectors will usually have 0 or 90 degrees most of the time.